BundID/TCA/Coordinator.swift

```diff
@@ -127,10 +108,6 @@ let coordinatorReducer: Reducer<CoordinatorState, CoordinatorAction, AppEnvironm
                         state.routes.dismiss()
                         return .none
                     }
-                case .setupCoordinator(.afterConfirmEnd),
-                        .identificationCoordinator(.afterConfirmEnd):
-                    state.routes.dismiss()
-                    return .none
                 case .setupCoordinator(.routeAction(_, action: .done(.triggerIdentification))):
```

Comment:
the triggerIdentification action comes with a tokenURL. Please use this one instead of the tokenURL of the state